Content/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.2-aspnetcore-runtime-alpine
+FROM microsoft/dotnet:3.0-aspnetcore-runtime-alpine
```

Comment:
This image does not seem to exist. Can't pull it from Docker Hub.

Reply:
The non-Alpine version is available.

Reply:
I think the naming scheme could have changed - looks like there's an alpine image here https://hub.docker.com/_/microsoft-dotnet-core-aspnet/

Reply:
That uses Microsoft's own registry. I'll use that if you're OK with it.

Reply:
I'm ok with either as long as it's an "official" image